### ISSUE TITLE
Replays Still look a tiny bit jumpy/sluggish

### ISSUE THREAD

**Issue by mattmoparkour** [Igrium/ReplayFPS#4]
Is there any way to make this mod work even better (store more info on the players position and stuff)? There's still some occasional sluggish jolting movement and it overally looks a tiny bit too floaty if ykwim. Im recording in 144fps vsync and rendering at 60 if that means anything. Ill provide an example if it's of any use.

**Comment by mattmoparkour** [comment]
Im using quite a few other performance related mods by the way. Could this be interfering with it?

**Comment by Igrium** [comment]
Can you send a video example? The mod's still in alpha, so it's very possible there's something I didn't account for.

**Comment by mattmoparkour** [comment]
There's a little bit of slowness but only occasionally (first appearing at 0:08). Here's [my mods folder](https://mega.nz/file/NbdEBaxY#DaaAAfLLPmFUypRy9RyW4OiWoEd-4Dio2jLFr1TbqS8) if it means anything.

https://mega.nz/file/0TUGhKDD#GC1IWNEvUQkB1UuQBshXxXAeYlFckV7Oc0GExPbfnIs

**Comment by mattmoparkour** [comment]
Also, just a quick question: would this mod have the same effect if i recorded using it then exported without it? For background info, I plan to use lunar time changer to change the time of gameplays however I cant get replayfps on lunar so i plan to record separately before rendering in lunar.

**Comment by Igrium** [comment]
To me, that looks like ReplayFPS failed to capture any data; what you see there is what the Replay Mod captures on its own. There's a known issue where, if you stop the replay recording through the menu *before* you exit the world, the client data will fail to save. Just leave the world and let the replay save automatically.

**Comment by mattmoparkour** [comment]
ohh bro thanks so much for telling me this

**Comment by mattmoparkour** [comment]
Here's what it looks like now. The black screen glitch is still there and when i turn in a 360 fast, it interpolates it which tells me that it might not be working(?). It definitely looks better than before I think. https://mega.nz/file/BC0ByCKC#9GgGjWG1CEjekpDk00oEWz_plmQM-FFHsFkRjX5VQXM

**Comment by Igrium** [comment]
Yeah, that's definitely not any better. Can you send the mcpr file? The mod doesn't seem to be activating at all.

Also, ReplayFPS doesn't touch the render system, so the black flicker is probably Replay Mod's fault.

**Comment by mattmoparkour** [comment]
Okay. This is a different recording because i deleted the other one, but it has the same problem. https://mega.nz/file/RTcxGbZY#Dh-EJTe4_eS712Da1Y3nzIBIhIeEs7Tx7NEJEqHNZFU

**Comment by mattmoparkour** [comment]
I managed to get it working! It seems to be an issue caused by my other mods, because when i made a new modrinth instance with only replayfps and replaymod, everything was perfect. Ill work up from there and see which mod it actually was. If you like I can let you know.

**Comment by mattmoparkour** [comment]
This also prevents the black screen glitch which is good

**Comment by mattmoparkour** [comment]
Turns out the issue is that automatic recording needs to be switched on. This mod also fixes the black screen glitch so thats great as well. Thanks for your help + i greatly appreciate you making this mod since it helps me out with recording parkour for clients. Lmk if you have a patreon or similar.

**Comment by mattmoparkour** [comment]
When i record in vysnc it looks a little choppy, but when i record with unlimited it looks really smooth. Do you know how i can fix this?

**Comment by mattmoparkour** [comment]
Do you know how I can make the camera movement stay smooth whilst keeping the actual player movement good. I like the camera easing in replaymod but replayfps prevents it. I understand there probably isn't a way around it, but i would like to keep the smoothness of the camera.
